tools/as/as.cpp

```diff
@@ -79,6 +84,15 @@ int main(int argc, char** argv) {
             return 1;
           }
         } break;
+        case '-': {
+          // Long options
+          if (0 == strcmp(argv[argi], "--version")) {
+            printf("%s\r\n", kBuildVersion);
```

Comment:
I remember on Windows '\r\n' is the newline, but actually I'm not sure whether we need to do this here.

Another thing that I feel weird is that why are we using 'printf' instead of 'std::count' in these spirv-tools?